### ISSUE TITLE
[Dex lives Issue] 

### ISSUE THREAD

**Issue by KungFuHustler** [ObliqueNetwork/Terra#2357]
when using a pokemon's live dex it uses 2 instead of 1 and if it only has 1 left it just deletes the dex live (uses 1 dex as usual when catching the legendary) i

**To Reproduce**
1)have Pokemon with Dex lives 
2)Pokegift or trade other person
3)should normally take 1 Dex lives but often takes 2

if it has 1 life left the life will disapear and it wont count for dex

Kung_Fu_Hustler

tried on atlas and titan double dexed on both

[techniclauncher_2024-01-16.log](https://github.com/NebulaMC-GG/Bugs-and-Issues/files/13954548/techniclauncher_2024-01-16.log)

for some reason it doesn't always take 2 and even sometimes gains 1 randomly

**Comment by josmit48** [comment]
Username: Temperestre
Server: Atlas
I traded pokemon with 1 dexlife left in the trade machine with Mitchkiller10 just now. My Enamorus lost its last life and Mitch did not get a dex entry. Mitch's zekrom was at 1 dex life left, then I did not get a dex entry for Zekrom.

No dex entry for Zekrom:
https://cdn.discordapp.com/attachments/1197439824970403961/1197440518074945566/image.png?ex=65bb4662&is=65a8d162&hm=2816dbbccffde9c613bc77064a67cc042c7dc5152bb04d34e9f2b50db8c15a39&

[latest.log](https://github.com/NebulaMC-GG/Bugs-and-Issues/files/13973400/latest.log)

P.S: I am using technic and Mitch was using Nebula launcher. (If that affects it)
